### PR TITLE
Taskcluster community migration

### PR DESCRIPTION
**Pull request by La0**
I stole some url changes from @imbstack and fixed some pre-commit issues.

Sorry i inadvertently closed the previous PR :cry:

### CODE REVIEW

**Comment by La0** [comment]
@marco-c The PR is now OK on TC-Community, i just switched the repo merge rules to require TC-Community check instead of Taskcluster. We should be good to go after your review

File: bot/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ code-review-bot --taskcluster-secret=path/to/secret
 Configuration
 -------------
 
-The code review bot is configured through the [Taskcluster secrets service](https://tools.taskcluster.net/secrets)
+The code review bot is configured through the [Taskcluster secrets service](https://community-tc.services.mozilla.com/secrets)
```
Comment (marco-c) [inline bot/README.md]: This should be firefox-ci

File: docs/new_repository.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Requirements
 
 1. Your repository must use [Mozilla Phabricator's instance](https://phabricator.services.mozilla.com/)
-2. Your repository must use [Taskcluster as CI](https://docs.taskcluster.net) (at least one task must start on each push).
+2. Your repository must use [Taskcluster as CI](https://community-tc.services.mozilla.com/) (at least one task must start on each push).
```
Comment (marco-c) [inline docs/new_repository.md]:
```suggestion
2. Your repository must use [Taskcluster as CI](https://community-tc.services.mozilla.com/docs) (at least one task must start on each push).
```

File: bot/README.md (hunk 1 above)
Comment (La0) [reply bot/README.md]: Well both... I guess the Heroku TC clients  will be on commuity-tc

File: tools/code_review_tools/taskcluster.py [hunk 3]
```diff
@@ -185,4 +185,4 @@ def create_blob_artifact(
     )
 
     # Build the absolute url
-    return f"https://queue.taskcluster.net/v1/task/{task_id}/runs/{run_id}/artifacts/{path}"
+    return f"https://community-tc.services.mozilla.com/api/queue/v1/task/{task_id}/runs/{run_id}/artifacts/{path}"
```
Comment (marco-c) [inline tools/code_review_tools/taskcluster.py]: This should be firefox-ci

File: tools/code_review_tools/taskcluster.py [hunk 4]
```diff
@@ -86,7 +86,7 @@ def auth(self, client_id=None, access_token=None):
                 "clientId": client_id,
                 "accessToken": access_token,
             }
-            self.options["rootUrl"] = "https://taskcluster.net"
+            self.options["rootUrl"] = "https://community-tc.services.mozilla.com"
```
Comment (marco-c) [inline tools/code_review_tools/taskcluster.py]: This should be firefox-ci, right?

File: tools/code_review_tools/taskcluster.py (hunk 3 above)
Comment (La0) [reply tools/code_review_tools/taskcluster.py]: Yes, this is used to upload patches from code-review bot tasks, so in firefox ci.

File: tools/code_review_tools/taskcluster.py (hunk 4 above)
Comment (La0) [reply tools/code_review_tools/taskcluster.py]: I would agree to firefox-ci here, this should be used only when providing directly TC credentials, meaning a code-review dev trying to debug a task.

File: bot/README.md (hunk 1 above)
Comment (marco-c) [reply bot/README.md]: It might be better to use firefox-ci for everything deployed, and community-tc only for CI, otherwise we have to duplicate secrets

Comment (La0) [reply bot/README.md]: Done

**Pull request merged**